examples/ERP/classify_P300_bi_illiteracy.py

```diff
@@ -0,0 +1,333 @@
+# but the evaluation function uses a LabelEncoder, transforming them
+# to 0 and 1
+labels_dict = {"Target": 1, "NonTarget": 0}
+
+
+##############################################################################
+# Create Pipelines
+# ----------------
+#
+# Pipelines must be a dict of sklearn pipeline transformer.
+
+
+class PIP(str, Enum):
+    xDAWNCov_TsLDA = "(C) XdawnCov+TsLDA"
+    xDAWN_LDA = "(C) Xdawn+LDA"
+    xDAWNCov_MDM = "(C) XdawnCov+MDM"
+    xDAWNCov_TsSVC = "(C) XdawnCov+TsSVC"
+    ERPCov_CvxMDM_Dist = "(C) ERPCov+CvxMDM+Dist"
+    ERPCov_QMDM_Dist = "(Q) ERPCov+QMDM+Dist"
+    xDAWNCov_TsQSVC = "(Q) XdawnCov+TsQSVC"
+    Vot_QMDM_Dist_Mean = "(Q) Vot+QMDM+Dist+Mean"
+    Vot_QMDM_MDM = "(Q) Vot+QMDM+MDM"
+    Judge_QMDM_MDM_TsLDA = "(Q) GradBoost_QMDM_MDM"
+
+
+pipelines = {}
+
+USE_PLACEHOLDERS = False
+
+
+def placeholder(key):
+    if USE_PLACEHOLDERS:
+        return
+    pipelines[key] = Pipeline(
+        steps=[
+            (
+                key,
+                XdawnCovariances(
+                    nfilter=4,
+                    classes=[labels_dict["Target"]],
+                    estimator="scm",  # add to classification?
+                    xdawn_estimator="lwf",
+                ),
+            ),
+            ("TS", TangentSpace()),
+            ("LDA", LDA(solver="lsqr", shrinkage="auto")),
+        ]
+    )
+
+
+## Classical Pipelines
+
+pipelines[PIP.xDAWNCov_TsLDA.value] = make_pipeline(
+    XdawnCovariances(
+        nfilter=4,
+        classes=[labels_dict["Target"]],
+        estimator="scm",  # add to classification?
+        xdawn_estimator="lwf",
+    ),
+    TangentSpace(),
+    LDA(solver="lsqr", shrinkage="auto"),
+)
+placeholder(PIP.xDAWNCov_TsLDA.value)
+
+pipelines[PIP.xDAWNCov_TsSVC.value] = make_pipeline(
+    XdawnCovariances(
+        nfilter=4,
+        classes=[labels_dict["Target"]],
+        estimator="scm",
+        xdawn_estimator="lwf",
+    ),
+    TangentSpace(),
+    SVC(),
+)
+placeholder(PIP.xDAWNCov_TsSVC.value)
+
+pipelines[PIP.xDAWNCov_MDM.value] = make_pipeline(
+    XdawnCovariances(
+        nfilter=4,
+        classes=[labels_dict["Target"]],
+        estimator="scm",
+        xdawn_estimator="lwf",
+    ),
+    MDM(),
+)
+placeholder(PIP.xDAWNCov_MDM.value)
+
+pipelines[PIP.xDAWN_LDA.value] = make_pipeline(
+    Xdawn(nfilter=3),
+    Vectorizer(),
+    LDA(solver="lsqr", shrinkage="auto"),
+)
+placeholder(PIP.xDAWN_LDA.value)
+
+pipelines[PIP.ERPCov_CvxMDM_Dist.value] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=False
+)
+placeholder(PIP.ERPCov_CvxMDM_Dist.value)
+
+## Quantum Pipelines
+
+pipelines[PIP.ERPCov_QMDM_Dist.value] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=True
+)
+placeholder(PIP.ERPCov_QMDM_Dist.value)
+
+pipelines[PIP.ERPCov_QMDM_Dist.value] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=True
+)
+placeholder(PIP.ERPCov_QMDM_Dist.value)
```

Comment:
Why is this pipeline called "ERPCov"?
because it does not use [ERPCovariances](https://pyriemann.readthedocs.io/en/latest/generated/pyriemann.estimation.ERPCovariances.html)

Reply:
Sorry, I hadn't seen the inside of the code QuantumMDMWithRiemannianPipeline.